app/components/osf-footer/component.ts

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import { serviceLinks } from 'ember-osf-web/const/service-links';
+
+export default class OsfFooter extends Component {
+    supportEmail = 'placeholder';
+    serviceLinks = serviceLinks;
+
+    didRender() {
```

Comment:
Apparently overriding `didRender` like this prevents a11y checking on this component. This sort of simple property setting should go in `constructor()`, anyway.